pre_commit_hooks/sort_cspell_json.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 Tier IV, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+from pathlib import Path
+
+
+# The key contained in the json.
+# The default setting is {unique: False, sort: False, key_order: None}
+KEYS = {
+    "ignorePaths": {"unique": True, "sort": True, "key_order": None},
+    "words": {"unique": True, "sort": True, "key_order": str.lower},
+}
+
+
+def unique_json(data, key):
+    if key in data:
+        data[key] = list(set(data[key]))
+    return data
+
+
+def sort_json(data, key, key_order=None):
+    if key in data:
+        data[key] = sorted(data[key], key=key_order)
+    return data
+
+
+def parse(data):
```

Comment:
For more readability.

```suggestion
def format_cspell_json(cspell_json):
```

Reply:
6157ca3